src/Augmentation/AugmentationHelpers.tsx

```diff
@@ -85,9 +85,17 @@ function applyAugmentation(aug: IPlayerOwnedAugmentation, reapply = false): void
     Player.applyEntropy(Player.entropy);
   }
 
+  // Special logic for NeuroFlux Governor
+  const ownedNfg = Player.augmentations.find((pAug) => pAug.name === AugmentationNames.NeuroFluxGovernor);
+  if (aug.name === AugmentationNames.NeuroFluxGovernor && !reapply && ownedNfg) {
```

Comment:
i wonder if this should be `(!reapply || ownedNfg)`

Reply:
lol nvm that wouldnt work

Reply:
i don't think so - when reapplied, the level from the save object is what's applied, so that's not a concern. if `ownedNfg` is undefined, then the special logic will be skipped and the player's first NFG should be applied